common-custom-user-data-maven-extension/pom.xml

```diff
@@ -65,7 +65,7 @@
         <dependency>
             <groupId>com.gradle</groupId>
             <artifactId>gradle-enterprise-maven-extension</artifactId>
-            <version>1.10.3</version>
+            <version>1.11</version>
```

Comment:
Will we have to wait for the 1.11 release before merging this PR?

Reply:
yes, we will have to wait for BA Maven 1.11 to be out first. This version will ship the new `GradleEnterpriseListener` interface, which is used instead of hooking ourselves into Maven lifecycle directly (via `AbstractMavenLifecycleParticipant` or event spies)